packages/airnode-node/src/handlers/call-api.ts

```diff
@@ -211,9 +211,30 @@ async function processSuccessfulApiCall(
     const signature = await signResponseMessage(aggregatedApiCall.id, response.encodedValue, config);
     return [[], { success: true, value: response.encodedValue, signature }];
   } catch (e) {
-    const data = JSON.stringify(rawResponse.data);
-    const log = logger.pend('ERROR', `Unable to find response value from ${data}. Path: ${reservedParameters._path}`);
-    return [[log], { success: false, errorMessage: RequestErrorMessage.ResponseValueNotFound }];
+    if (e instanceof adapter.MissingValueError) {
```

Comment:
I think we can assume that the `e` will always be and `Error`. And I think this checking is overkill, since we ultimately only care about the user receiving a good error message.

Reply:
This could be changed to just pass the `error.message` into both the logger `message` and `errorMessage` fields. But in this case how would we specify `RequestErrorMessage` enums?